creator/table_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"io/ioutil"
 	"testing"
 
+	"github.com/stretchr/testify/require"
```

Comment:
fix import style, usually would have one line in between

Reply:
Done.